src/components/Home/Home.tsx

```diff
@@ -17,7 +17,7 @@ const Component: React.SFC<IDispatchProps & IStateProps> = ({
   user
 }) => (
     <Content className="rvt-p-top-xl rvt-bg-white rvt-p-bottom-xl rvt-m-top-xxl rvt-m-bottom-xxl">
-      <PageTitle>IT People</PageTitle>
+      <PageTitle data-testid="home-page">IT People</PageTitle>
```

Comment:
What is `data-testid="home-page"` used for?

Reply:
pretty much what it says- it creates an id in the document for tests to find. It's a fallback technique when there isn't another way to identify the component.